public/styles.css

```diff
@@ -75,27 +105,17 @@ button:hover{
     -webkit-background-clip: text;
     background-color: transparent;
     overflow:auto;
+    color: white;
+    font-family:sans-serif;
 }
 .text-container{
     background-color: transparent;
 }
 
 html:hover{
-    background-color:#5AC87ED9;
-    /*animation: blink-animation 0.3s steps(5, start) infinite;
-    -webkit-animation: blink-animation 0.3s steps(5, start) infinite; */
-  }
-  /* @keyframes blink-animation {
-    to {
-        background-color:rgba(46, 78, 255, 0.89);
-    }
-  }
-  @-webkit-keyframes blink-animation {
-    to {
-        background-color:rgba(240, 54, 54, 0.89);
-    }
-  } */
-
+    background-color:#f0e9ea;
```

Comment:
Too bright, make it change color to a more subtle one, no need to go from dark to bright like that